prover/src/channel.rs

```diff
@@ -68,6 +68,14 @@ where
         }
     }
 
+    // GETTER METHODS
+    // --------------------------------------------------------------------------------------------
+
+    /// Returns the inner public coin
+    pub fn public_coin(&mut self) -> &mut R {
+        &mut self.public_coin
+    }
```

Comment:
minor nit: I would probably move this into the `PUBLIC COIN METHODS` section below.

Reply:
fixed